docs/developer/images.md

```diff
@@ -0,0 +1,128 @@
+# Art and Images
+
+## Colors
+
+Images in arcade are stored using 4 bits per color. The color `0` is
+reserved for transparency, so there are 15 available colors for you to
+work with at any one time. Colors are represented as indices into a palette;
+the default palette shown in the MakeCode UI is here:
+
```

Comment:
Maybe something like:

Images in arcade are stored as rows and columns of color values. The values are 4 bits and allow for 16 different colors. The color `0` is reserved for transparency so, at any one time, you have the remaining 15 colors available to work with. The image color values are really indices that map to 16 entries of the current palette of RGB colors. The default palette for the MakeCode UI is here: